next.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   i18n: {
-    locales: ['en', 'pt'],
+    locales: ['en', 'pt_BR'],
```

Comment:
The correct is pt-BR (with hiphen)

Reply:
Okay, I'll update that in the https://github.com/w3b3d3v/buildspace-projects as well